src/main/java/com/sap/oss/phosphor/fosstars/advice/oss/OssAdviceContentYamlStorage.java

```diff
@@ -34,8 +34,12 @@
 
   static {
     try {
+      // FIXME (mibo): check if this could be done for all in OssFeatures defined
+      //                features instead of being based on a rating
       DEFAULT = OssAdviceContentYamlStorage.loadFromResource(
-          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssSecurityRating.class));
+          RESOURCE_PATH, RatingRepository.INSTANCE.rating(OssArtifactSecurityRating.class));
```

Comment:
When it is reverted to `OssSecurityRating` the additional features of the `OssArtifactSecurityRating` will not be found.
My recommendation would be to keep it for now with `OssArtifactSecurityRating` as this rating contains all required features.
In issue #474 it can then be change to get the features e.g. somehow from the `OssFeatures`.

Reply:
In this method `com.sap.oss.phosphor.fosstars.advice.AdviceContentYamlStorage#loadFromResource` it loads all features based on the given rating.
If the ratings does not contains all features and the CLI is used below exception is thrown:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at com.sap.oss.phosphor.fosstars.advice.oss.CodeqlAdvisor.<init>(CodeqlAdvisor.java:26)
	at com.sap.oss.phosphor.fosstars.advice.oss.github.OssSecurityGithubAdvisor.<init>(OssSecurityGithubAdvisor.java:56)
	at com.sap.oss.phosphor.fosstars.tool.github.Application.<clinit>(Application.java:115)
Caused by: java.io.UncheckedIOException: Could not load advice
	at com.sap.oss.phosphor.fosstars.advice.oss.OssAdviceContentYamlStorage.<clinit>(OssAdviceContentYamlStorage.java:41)
	... 3 more
Caused by: java.io.IOException: Could not find feature in the rating: Artifact version
	at com.sap.oss.phosphor.fosstars.advice.AdviceContentYamlStorage.loadFromResource(AdviceContentYamlStorage.java:106)
	at com.sap.oss.phosphor.fosstars.advice.oss.OssAdviceContentYamlStorage.loadFromResource(OssAdviceContentYamlStorage.java:82)
	at com.sap.oss.phosphor.fosstars.advice.oss.OssAdviceContentYamlStorage.<clinit>(OssAdviceContentYamlStorage.java:38)
	... 3 more
```

Reply:
I see. Let's then keep `OssArtifactSecurityRating` here and open an issue to address this later.